README.md

```diff
@@ -53,8 +53,8 @@ The ConnectorConfig.json file can be edited through the configuration webpage.
 #### FTP Configuration
 If you wish to manually edit your ConnectorConfig.json file or you have a saved ConnectorConfig.json file from another Flexy you can load your ConnectorConfig.json file on the flexy in the /usr/ directory via FTP.
 
-### Flexy Clock Settings
-The Flexy's clock must be set to Coordinated Universal Time (UTC) in order to post accurate timestamps to OSIsoft.  This can be done manually on the Flexy's websever under Setup->System->Main->General->Date & Time or NTP can be used if there is an internet connection available.  NTP is setup in Setup->System->Main->Net services->NTP (time).  If using NTP time make sure the GMT Offset is set to zero hours
+### Flexy Configuration
+The Flexy must have a FTP user created to be used by the application in order to post accurate timestamps to OSIsoft. This can be done manually on the Flexy's websever under Setup->Users page through the "Add" button. The added user login should be "FtpUser" and the added user password should be "FtpPassword". Check all global rights option check boxes.
```

Comment:
If there is no way for the user to change the default of FtpUser/FtpPassword, then I think those should be changed. At least the password should be stronger and maybe include some capitals/numbers.

Reply:
In the other connectors it makes more sense to not have user configurable username/pass combos as those sources are meant to be changed by the user. For this connector it would probably make more sense to have that configured in the config file or somewhere more secure.

Reply:
fixed